src/nuget/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-featmsbuild-*",
```

Comment:
With this new project, are we getting a new .dll named "nuget.dll" in the output folder?  if yes, is that intentional?